pkg/oci/ccm.go

```diff
@@ -130,24 +129,22 @@ func (cp *CloudProvider) Initialize(clientBuilder controller.ControllerClientBui
 
 	nodeInformer := factory.Core().V1().Nodes()
 	go nodeInformer.Informer().Run(wait.NeverStop)
+	serviceInformer := factory.Core().V1().Services()
+	go serviceInformer.Informer().Run(wait.NeverStop)
```

Comment:
Might want to stick a `\n` under here to separate out the setup vs the cache sync